pages/common/nslookup.md

```diff
@@ -0,0 +1,27 @@
+# nslookup
+
+> Query name server(s) for various domain records.
+
+- Query your system's default name server for an IP address (A record) of the domain:
+
+`nslookup {{example.com}}`
+
+- Query non default name server (here Google DNS 8.8.8.8) for a NS record of the domain:
+
+`nslookup -type=NS {{example.com}} 8.8.8.8`
+
+- Query default name server for a reverse lookup (PTR record) of an IP address:
+
+`nslookup -type=PTR 54.240.162.118`
+
+- Query a given name server for the whole zone file (zone transfer) of the domain using TCP protocol (-vc option) instead of the default UDP:
+
+`nslookup -vc -type=AXFR {{example.com}} {{name_server}}`
```

Comment:
I think you can introduce -vc separately, which would allow the example description to be less verbose. (When you introduce an example that uses an option that has been used in a previous example, you don't need to describe it as extensively.)